_commands/tools/bzip2.md

```diff
@@ -0,0 +1,44 @@
+---
+---
+
+bzip2
+-------
+<!--TODO: Add documentation for this command by submitting a pull request.-->
+<!-- one line explanation would go here -->
+`bzip2` is the command used for compressing and decompressing files.
+
+<!-- minimal example -->
+~~~ bash
+$ bzip2 filename
+~~~
+
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `bzip2`
+
+~~~bash
+$ bzip2 [Option(s)] filename(s)
+~~~
+
+Options include:
+
+1. `-v` for verbose: provide confirmation and degree to which the data is compressed
+2. `-s` for small: reduce the memory usage for compression
+3. `-d` for decompress: decompress the specified file
+
+----
+
+#### `bzip2 file1 file2 file3`
+
+~~~bash
+$ bzip2 file1 file2 file3
+~~~
+
+##### Break it down
+
+*Compress multiple files (file1 file2 and file3, in this case) in the "zip" format at the same time. Each file is replaced by a compressed version of itself, with the extension .bz2 appended to its name. Thus, in the case of the above example, the three input files would be replaced by files named file1.bz2, file2.bz2 and file3.bz2. This can be easily confirmed with the ls (i.e., list) command, and the sizes of the new files can be viewed by using ls together with its -s (i.e., size) option. The original files can be retained by using the -k (i.e., keep) option.
```

Comment:
I think the asterisk at the start of this paragraph is not needed, so you can remove that.

Reply:
asterisk removed